indexer/migrations/20230523_create_schema.sql

```diff
@@ -84,6 +84,31 @@ CREATE TABLE IF NOT EXISTS output_proposals (
  * BRIDGING DATA
  */
 
+ /**
+ * TOKEN DATA
+ */
+
+-- L1 Token table
+CREATE TABLE IF NOT EXISTS l1_tokens (
+    address   VARCHAR PRIMARY KEY,
+    bridge_address  VARCHAR NOT NULL,
+    l2_token_address VARCHAR NOT NULL,
```

Comment:
this relationship is 1->many so we should remove this column from `l2_tokens`.

Reply:
oh yea good call
```suggestion
```